package.json

```
@@ -11,6 +11,7 @@
     "watch": "tsc -w",
     "lint": "eslint . --rulesdir utils/eslint-rules",
     "lint:fix": "npm run lint -- --fix",
+    "standalone-package": "npm run clean && npm install && npm run build && npm-pack-all",
```

nit: Also, always good to keep npm install out of the npm scripts 

```suggestion
    "pkg": "npm run clean && npm run build && npm-pack-all",
```